script/bootstrap

```diff
@@ -9,14 +9,14 @@ fi
 STATUS=0
 
 if ! go version; then
-  echo "You need to install Go 1.2 to build hub" >&2
+  echo "You need to install Go 1.4.1 or higher to build hub" >&2
   STATUS=1
 fi
 
 { ruby --version
   bundle install --path vendor/bundle -j 4
 } || {
-  echo "You need Ruby 2.1 and Bundler to run hub tests" >&2
+  echo "You need Ruby 1.9 or higher and Bundler to run hub tests" >&2
```

Comment:
This is to make [`CONTRIBUTING.md`](https://github.com/github/hub/blob/master/CONTRIBUTING.md) and this script consistent - it caused confusion like #797. I don't have a problem of suggesting Ruby 2.1 if that works better. But it'd be nice that our build system (Travis) is running the same version.